addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -74,7 +74,10 @@ def _compute_web_page_link(self):
 
     @api.depends('channel_ids')
     def _compute_nbr_channel(self):
-        data = self.env['mail.channel'].read_group([('livechat_channel_id', 'in', self._ids)], ['__count'], ['livechat_channel_id'], lazy=False)
+        data = self.env['mail.channel'].read_group([
+            ('livechat_channel_id', 'in', self._ids),
+            ('channel_message_ids', '!=', False)
```

Comment:
I fear this small addition will be difficult to compute: first go into a crowded m2m between messages and channels, then return an [(id, in, found_ids)] where foud_ids can be quite huge when using a lot of livechat.

Isn't it easier to fetch channels, then filter out those without messages ?